plugins/fixed-edges/src/index.js

```diff
@@ -72,19 +72,19 @@ export class FixedEdgesMetricsManager extends Blockly.MetricsManager {
 
   /**
    * Computes the fixed edges of the scroll area.
-   * @param {!Blockly.MetricsManager.ContainerRegion=} opt_viewMetrics The view
+   * @param {!Blockly.MetricsManager.ContainerRegion=} viewMetricsParam The view
```

Comment:
This is marked as non-nullable type should i change it to `?(Blockly.MetricsManager.ContainerRegion|undefined)=` or keep it non-nullable and change it to `!(Blockly.MetricsManager.ContainerRegion|undefined)=`

Reply:
For now i have changed it to `!(Blockly.MetricsManager.ContainerRegion|undefined)=`

Reply:
For [optional parameters](https://github.com/google/closure-compiler/wiki/Types-in-the-Closure-Type-System#optional) you do not actually need the `|undefined` since the `=` denotes that it is optional and therefore can be undefined. The `|undefined` is only needed when defining property types. More documentation on this can be found in the [Optional Parameters and Properties section](https://github.com/google/closure-compiler/wiki/Types-in-the-Closure-Type-System#nullable-vs-optional-parameters-and-properties). When I mentioned adding `|undefined` it was just for [this line](https://github.com/google/blockly-samples/blob/e024b9e53a4d0f6d45f3e9103bf9503fb194f0eb/plugins/block-plus-minus/src/field_minus.js#L24) and [this line](https://github.com/google/blockly-samples/blob/e024b9e53a4d0f6d45f3e9103bf9503fb194f0eb/plugins/block-plus-minus/src/field_plus.js#L24) all the optional parameters can be reverted so they only have the `=` sign. Sorry for the confusion! And let me know if you have any other questions.

Reply:
Oh! Sorry i thought you were talking about `@param`  line